modules/cloud-provider-equinix-metal/README.md

```diff
@@ -0,0 +1,72 @@
+## Cloud Provider Equinix Metal
+
+<!-- TEMPLATE: Review all "TEMPLATE" comments and remove them when applied. -->
+<!-- TEMPLATE: replace "template" with the name of your project. The prefix "terraform-equinix-" informs the Terraform registry that this project is a Terraform module associated with the Equinix provider, Oreserve this prefix.  "terraform-metal-" may also be used for Equinix Metal modules, but "terraform-equinix-" will work too. -->
+[![Experimental](https://img.shields.io/badge/Stability-Experimental-red.svg)](https://github.com/equinix-labs/standards#about-uniform-standards)
+[![terraform](https://github.com/equinix-labs/terraform-equinix-template/actions/workflows/integration.yaml/badge.svg)](https://github.com/equinix-labs/terraform-equinix-template/actions/workflows/integration.yaml)
+
+`cloud-provider-equinix-metal` is a Terraform module that installs and configures Cloud Provider Equinix Metal on a kubernetes cluster.
+
+<!-- TEMPLATE: Insert an image here of the infrastructure diagram. You can generate a starting image using instructions found at https://www.terraform.io/docs/cli/commands/graph.html#generating-images -->
+
+### Usage
+
+This project is experimental and supported by the user community. Equinix does not provide support for this project.
+
+Install Terraform using the official guides at <https://learn.hashicorp.com/tutorials/terraform/install-cli>.
+
+This project may be forked, cloned, or downloaded and modified as needed as the base in your integrations and deployments.
+
+This project may also be used as a [Terraform module](https://learn.hashicorp.com/collections/terraform/modules).
+
+To use this module in a new project, create a file such as:
+
+```hcl
+# main.tf
+terraform {
+  required_providers {
+    equinix = {
+      source = "equinix/equinix"
+    }
+    metal = {
+      source = "equinix/metal"
```

Comment:
```suggestion
```

(and remove the `metal = {}` -- I assume we don't actually need that here (or in any new Terraform modules) and this was copied from the Template.